src/server.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import { Provider } from 'react-redux';
+import { StaticRouter as Router } from 'react-router-dom';
+
+import configureStore from 'configureStore.js';
+import AppView from 'views/AppView.jsx';
+
+export default function (props) {
+  const store = configureStore();
+  const markup = ReactDOMServer.renderToString(
+    <Provider store={store}>
+      <Router location={props.req.url} context={props.context} ><AppView /></Router>
+    </Provider>,
+  );
```

Comment:
Generate the markup, we are using a StaticRouter, which is a Router in Memory that works in the server, we need the URL of the request to send the right router of the app